rotkehlchen/exchanges/bitstamp.py

```diff
@@ -729,7 +729,7 @@ def _deserialize_asset_movement_from_user_transaction(
             unique_id=(movement_id := str(raw_movement['id'])),
             extra_data={
                 'movement_id': movement_id,
-                'fee': fee,  # Used for matching the corresponding crypto_transaction. Removed before being saved to the DB.  # noqa: E501
+                'fee': fee,
```

Comment:
This is used here: https://github.com/rotki/rotki/blob/27d664cae869c24e721a36060805292149aaead7/rotkehlchen/exchanges/bitstamp.py#L300 to match the events from the crypto transaction endpoint, which include the fee in their amount. There not really any point in saving this to the database since this is the only place its used, so then I have it being removed just below: https://github.com/rotki/rotki/blob/27d664cae869c24e721a36060805292149aaead7/rotkehlchen/exchanges/bitstamp.py#L304 But this is rather hacky and there's a note about the logic in this exchange needing improvement: https://github.com/orgs/rotki/projects/11/views/2?pane=issue&itemId=90720824